app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+
+  def recent_posts
+    posts.order(created_at: :desc).limit(3)
+  end
+end
```

Comment:
[OPTIONAL] Great job setting up your models! Just a tip, the foreign keys are already associated in this user.rb model, you don't need to associate them again post.rb, comment.rb or like.rb.